src/Pages/FAQ/index.js

```diff
@@ -0,0 +1,50 @@
+import styled from "styled-components";
+
+const FAQ = () => {
+  const faq = [
+    ["Quality:", "Our burgers are the best!"],
+    ["Price", "Щur price is the lowest!"],
+    ["Our Service", "Our service is the fastest in the world!"],
+  ];
+  return (
+    <Wrapper>
+      <Tittle>FAQ</Tittle>
+      <ListStyled>
+        {faq.map((el) => {
+          return (
+            <ItemStyled key={el[0]}>
+              <SpanStyled>{el[0]} : </SpanStyled>
+              <SpanStyled>{el[1]}</SpanStyled>
+            </ItemStyled>
+          );
+        })}
+      </ListStyled>
+    </Wrapper>
```

Comment:
As for me, simple HTML elements look like the easiest solution, there are plenty of options in UI libraries